pkgs/build-support/vm/linuxkit-builder/kernel.nix

```diff
@@ -0,0 +1,10 @@
+{ pkgs }:
+
+(pkgs.linuxPackages_custom {
+  inherit (pkgs.linux_4_9) src;
+  version = "${pkgs.linux_4_9.version}-linuxkit";
+  configfile = pkgs.fetchurl {
+    url = https://raw.githubusercontent.com/linuxkit/linuxkit/cb1c74977297b326638daeb824983f0a2e13fdf2/kernel/kernel_config-4.9.x-x86_64;
```

Comment:
Maybe a comment about how you picked this version and what someone might want to do to update it?